.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  root: true,
+
+  env: {
+    node: true,
+  },
+
+  extends: [
+    'plugin:vue/recommended',
+    '@vue/airbnb',
+    '@vue/typescript',
+    'plugin:vue/recommended',
+    '@vue/typescript/recommended',
+  ],
+
+  plugins: [
+    'vuetify',
+  ],
+
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'max-len': ['off'],
```

Comment:
Does the formatter attempt to break long lines? If it does, I would set this to something like 100 so that the formatter at least has a chance to simplify our code. I like the situation where, if the formatter can only break up your long line by making it harder to read, it's a signal to you to refactor a bit for readability. If you allow unlimited line length, you lose this benefit.

Reply:
I think this was in the default .eslintrc.js. It's also in our mutlinet-client repo so keeping it allows us to maintain parity with that config.

Reply:
Opened #117